src/Layout.tsx

```diff
@@ -1,78 +1,20 @@
 import { useEffect, useState } from "react";
 import { Outlet, useSearchParams } from "react-router-dom";
-import { nanoid } from "nanoid";
 import randomWords from "random-words";
 import { configureAbly } from "@ably-labs/react-hooks";
 import InfoCard from "./InfoCard";
-import { Types } from "ably";
-import { SignJWT } from "jose";
+import SpacesInfoCard from "./SpacesInfoCard";
+import type { ProjectInfo } from "./components/commonUtils/types";
+import apiConfig from "./components/commonUtils/apiConfig";
```

Comment:
I'd suggest that we create a folder on the same level as components, called `common` and put the `apiConfig` file there. I'd also suggest we use a verb here to better represent what this method is doing, something like `createAPIConfig`